compose/config/config_schema_v2.0.json

```diff
@@ -191,6 +191,7 @@
         "restart": {"type": "string"},
         "security_opt": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
         "shm_size": {"type": ["number", "string"]},
+        "sysctls": {"$ref": "#/definitions/list_or_dict"},
```

Comment:
Are there any references with more information about v2.1? I haven't been able to find much on the differences between v2.0 and v2.1.